lib/filters/index.js

```diff
@@ -139,6 +156,12 @@ module.exports = ruleSource => {
         }
       }
 
+      if (headerFilters.length) {
```

Comment:
very nitpick: the code is expecting an array so `Array.isArray(headerFilters)` is more correct